src/main/java/org/dbflute/intro/app/web/playsql/migration/PlaysqlMigrationAction.java

```diff
@@ -31,11 +34,16 @@
     //                                                                           Attribute
     //                                                                           =========
     @Resource
-    private PlaysqlMigrationLogic playsqlMigrationLogic;
+    private FlutyFileLogic flutyFileLogic;
+    @Resource
+    private IntroPhysicalLogic introPhysicalLogic;
 
+    // ===================================================================================
+    //                                                                             Execute
+    //                                                                             =======
     @Execute(urlPattern = "{}/@word")
     public JsonResponse<Void> alter(String clientName) {
-        playsqlMigrationLogic.openAlterDir(clientName);
+        flutyFileLogic.openDir(clientName, new File(introPhysicalLogic.buildClientPath(clientName, "playsql", "migration")));
```

Comment:
Keep openAlterDir() to manage physical logic by Logic.
PlaysqlMigrationLogic can use FlutyFileLogic.

Reply:
@jflute 
Reverted.
78f6eae6470ec17fb80674a8a7f974f7da62db9d